client/index.html

```diff
@@ -34,6 +34,7 @@
                         <legend>Selected object</legend>
                         <label class="selected__name"></label>
                         <button class="selected__drop-button" disabled>Drop</button>
```

Comment:
This is kind of misleading now, should probably change button text to "unselect"